gradle.properties

```diff
@@ -18,6 +18,7 @@ stdlibTimeVersion=2.2.4
 stdlibUrlVersion=2.2.3
 stdlibXmldataVersion=2.4.3
 observeVersion=1.0.7
+stdlibMathVectorVersion=1.0.1-20230529-131300-5c49593
```

Comment:
This should be the `1.0.0` version, which is released with 2201.4.x release. The next one will be released on top of the 2201.6.x release, which might not be compatible with 4.x releases.

Reply:
Ack. Addressed in https://github.com/ballerina-platform/ballerina-distribution/pull/4502/commits/3607d17d29e5e8c1a767fe212bb5ee513c3b3e9f.